Pop-Grupa-A-Backend/app/main/controller/task_controller.py

```diff
@@ -0,0 +1,50 @@
+from flask import request
+from flask_restplus import Resource
+from ..util.dto import TaskDto, StatusDto
+from ..repositories.task_repository import *
+from app.main.model.ComputationTask import ComputationTask
+
+
+api = TaskDto.api
+_task = TaskDto.task
+_status = StatusDto.status
+
+@api.route('/')
+class TaskList(Resource):
+    @api.response(201, 'Task successfully created.')
+    @api.doc('create a new task')
+    @api.expect(_task, validate=True)
+    def post(self):
+        """Creates a new Task"""
```

Comment:
Nazwa metody: `TaskList`
Opis: `Creates a new task`
zwraca nowy task

Chyba coś się tutaj nie zgadza....

Reply:
A to mój błąd, poprawione na moim branch'u. @anvime możesz to domergować z mojej gałęzi.